keyboards/varanidae/info.json

```diff
@@ -0,0 +1,151 @@
+                { "label":"K023", "matrix": [1, 3], "x": 3, "y": 1, "w": 1 },
+                { "label":"K024", "matrix": [1, 4], "x": 4, "y": 1, "w": 1 },
+                { "label":"K025", "matrix": [1, 5], "x": 5.25, "y": 1.25, "w": 1 },
+                { "label":"K026", "matrix": [1, 6], "x": 6.25, "y": 1.25, "w": 1 },
+                { "label":"K027", "matrix": [1, 7], "x": 7.25, "y": 1.25, "w": 1 },
+                { "label":"K028", "matrix": [1, 8], "x": 8.25, "y": 1.25, "w": 1 },
+                { "label":"K029", "matrix": [1, 9], "x": 9.25, "y": 1.25, "w": 1 },
+                { "label":"K030", "matrix": [6, 5], "x": 10.25, "y": 1.25, "w": 1 },
+                { "label":"K031", "matrix": [6, 6], "x": 11.25, "y": 1.25, "w": 1 },
+                { "label":"K032", "matrix": [6, 7], "x": 12.25, "y": 1.25, "w": 1 },
+                { "label":"K033", "matrix": [6, 8], "x": 13.25, "y": 1.25, "w": 1 },
+                { "label":"K034", "matrix": [6, 9], "x": 14.25, "y": 1.25, "w": 1 },
+                { "label":"K035", "matrix": [8, 0], "x": 15.25, "y": 1.25, "w": 1 },
+                { "label":"K036", "matrix": [8, 1], "x": 16.25, "y": 1.25, "w": 1 },
+                { "label":"K037", "matrix": [8, 2], "x": 17.25, "y": 1.25, "w": 1 },
+                { "label":"K038", "matrix": [8, 3], "x": 18.25, "y": 1.25, "w": 1 },
+                { "label":"K039", "matrix": [7, 4], "x": 19.25, "y": 1.25, "w": 1 },
+
+                { "label":"K040", "matrix": [2, 0], "x": 0, "y": 2, "w": 1 },
+                { "label":"K041", "matrix": [2, 1], "x": 1, "y": 2, "w": 1 },
+                { "label":"K042", "matrix": [2, 2], "x": 2, "y": 2, "w": 1 },
+                { "label":"K043", "matrix": [2, 3], "x": 3, "y": 2, "w": 1 },
+                { "label":"K044", "matrix": [2, 4], "x": 4, "y": 2, "w": 1 },
+                { "label":"K045", "matrix": [2, 5], "x": 5.25, "y": 2, "w": 1.5 },
+                { "label":"K046", "matrix": [2, 6], "x": 6.75, "y": 2, "w": 1 },
+                { "label":"K047", "matrix": [2, 7], "x": 7.75, "y": 2, "w": 1 },
+                { "label":"K048", "matrix": [2, 8], "x": 8.75, "y": 2, "w": 1 },
+                { "label":"K049", "matrix": [2, 9], "x": 9.75, "y": 2, "w": 1 },
+                { "label":"K050", "matrix": [7, 5], "x": 10.75, "y": 2, "w": 1 },
+                { "label":"K051", "matrix": [7, 6], "x": 11.75, "y": 2, "w": 1 },
+                { "label":"K052", "matrix": [7, 7], "x": 12.75, "y": 2, "w": 1 },
+                { "label":"K053", "matrix": [7, 8], "x": 13.75, "y": 2, "w": 1 },
+                { "label":"K054", "matrix": [7, 9], "x": 14.75, "y": 2, "w": 1 },
+                { "label":"K055", "matrix": [9, 0], "x": 15.75, "y": 2, "w": 1 },
+                { "label":"K056", "matrix": [9, 1], "x": 16.75, "y": 2, "w": 1 },
+                { "label":"K057", "matrix": [9, 2], "x": 17.75, "y": 2, "w": 1 },
+                { "label":"K058", "matrix": [9, 3], "x": 18.75, "y": 2, "w": 1.5 },
```

Comment:
Something's not quite right — looks like you need to adjust the `y` coordinates for many keys by `0.25`:
![Layout as shown on config.qmk.fm](https://github.com/qmk/qmk_firmware/assets/616082/ccc9a007-a6b7-4a0b-83a8-c5b86c37837b)
(To see the representation of your layout, you need to open https://config.qmk.fm, click once near the keyboard layout that appears to set the input focus properly, then press Ctrl+Shift+I and upload your `info.json` file.  Also switch legends to “Index” in the settings to confirm that the key ordering is correct; you may also use the “Matrix” setting to check the matrix row,col numbers.)

Reply:
Oh, you are right, thanks.